<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarMakerClass</class>
 <widget class="QMainWindow" name="StarMakerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarMaker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>StarMaker.ico</normaloff>StarMaker.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascensión Recta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Declinación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSegundos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Distancia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledistancia">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Distancia de la estrella en años luz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lemagnitud">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_appmag">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>AppMag</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_absmag">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>AbsMag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnitud</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo Espectral</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tespectral">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btncrear">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlimpiar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsalir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Estrella</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenomestrella">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHoras_ar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMinutos_ar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSegundos_ar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMinutos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHoras">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkParsec">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Si está activo, convierte los parsec en años luz</string>
    </property>
    <property name="text">
     <string>Parsec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leradio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRadioSol">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Si está activo, multiplicará el radio de la estrella (por ejemplo 1.54) por el radio del Sol</string>
    </property>
    <property name="text">
     <string>RSol</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Otros Nombres</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leotrosnombres">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Opcional. Nombres alternativos para la estrella, si es más de uno separar por coma</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionPlanetas"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionStarMaker"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStarMaker">
   <property name="text">
    <string>StarMaker</string>
   </property>
  </action>
  <action name="actionPlanetas">
   <property name="text">
    <string>Planetas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbHoras_ar</tabstop>
  <tabstop>cbMinutos_ar</tabstop>
  <tabstop>leSegundos_ar</tabstop>
  <tabstop>cbHoras</tabstop>
  <tabstop>cbMinutos</tabstop>
  <tabstop>leSegundos</tabstop>
  <tabstop>ledistancia</tabstop>
  <tabstop>chkParsec</tabstop>
  <tabstop>lemagnitud</tabstop>
  <tabstop>rb_appmag</tabstop>
  <tabstop>rb_absmag</tabstop>
  <tabstop>le_tespectral</tabstop>
  <tabstop>leradio</tabstop>
  <tabstop>chkRadioSol</tabstop>
  <tabstop>lenomestrella</tabstop>
  <tabstop>leotrosnombres</tabstop>
  <tabstop>btncrear</tabstop>
  <tabstop>btnlimpiar</tabstop>
  <tabstop>btnsalir</tabstop>
 </tabstops>
 <resources>
  <include location="starmaker.qrc"/>
 </resources>
 <connections/>
</ui>
